docs/en/operation/User-CLI.md

```diff
@@ -584,9 +584,19 @@ For example, `cp` can be used to copy files between under storage systems.
 $ ./bin/alluxio fs cp /hdfs/file1 /s3/
 ```
 
+### distributedCp
+
+The `distributedCp` command copies a file or directory in the Alluxio file system distributed across workers.
+
+If the source designates a directory, `distributedCp` copies the entire subtree at source to the destination.
+
+```console
+$ ./bin/alluxio fs distributedCp /data/1023 /data/1024
+```
+
 ### distributedLoad
 
-The `distributedLoad` command moves a file or directory from the under storage system into Alluxio storage in distributed 
+The `distributedLoad` command moves a file or directory from the under storage system into Alluxio storage distributed 
```

Comment:
Can we use a better word than "moves"? The implication being the data is then no longer in the under storage system. Maybe just "load"?